segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -355,9 +356,14 @@ private void handleException(UUID writerId, long requestId, String segment, long
             log.warn("Bad attribute update by {} on segment {}.", writerId, segment, u);
             connection.send(new InvalidEventNumber(writerId, requestId, clientReplyStackTrace));
             connection.close();
+        } else if (u instanceof TokenExpiredException) {
+            log.warn("Token was found as expired when writer {} was writing on segment {}", writerId, segment, u);
```

Comment:
"Token expired for writer {} on segment {}."

Reply:
Done.